### ISSUE TITLE
All core output has same time stamp

### ISSUE THREAD

**Issue by jcoffland** [FoldingAtHome/fah-client-pub#50]
**Reported by @jcoffland on 19 Jun 2010 21:34 UTC**
```
16:59:15:Unit 00:- Digital signature verified
16:59:15:Unit 00:
16:59:15:Unit 00:Project: 6014 (Run 0, Clone 35, Gen 210)
16:59:15:Unit 00:
16:59:15:Unit 00:Assembly optimizations on if available.
16:59:15:Unit 00:Entering M.D.
16:59:15:Unit 00:Completed 0 out of 500000 steps (0%)
16:59:15:Unit 00:Completed 5000 out of 500000 steps (1%)
16:59:15:Unit 00:Completed 10000 out of 500000 steps (2%)
16:59:15:Unit 00:Completed 15000 out of 500000 steps (3%)
16:59:15:Unit 00:Completed 20000 out of 500000 steps (4%)
```

**Comment by jcoffland** [comment]
**Comment by @jcoffland on 19 Jun 2010 21:40 UTC**
Fixed in v7.0.7

**Comment by jcoffland** [comment]
**Comment by @jcoffland on 23 Jun 2010 22:19 UTC**
Fixed in v7.0.7
